### DISCUSSION TITLE
Doc is not updating!

### DISCUSSION THREAD

**Discussion by irorochad** [fuma-nama/fumadocs#921] [Q&A] [answered]
Hey, 
Good day. I've been writing some docs update, it's working well locally. However, after I push and deploy to GitHub, the file isn't updated - some pages are not found.

Here's the repo link: [https://github.com/irorochad/harmono-docs](https://github.com/irorochad/harmono-docs ). 

You can view the live link here -> [https://harmono-docs-k2dxoyrb9-irorochads-projects.vercel.app/](https://harmono-docs-k2dxoyrb9-irorochads-projects.vercel.app/)

The scanner has a folder called `sdk` and it's supposed to show the `sdk` content, but it's not working..

Thank you!

**Comment by fuma-nama** [comment]
The url you posted is a private URL to a specific commit, you should use the production url of your Vercel app

**Reply by irorochad** [reply]
Sorry about that! [Here's the docs URL](https://docs.tryharmono.com/)

**Comment by fuma-nama** [comment, marked as answer]
It's because the case of file names on Linux matters, you need to use lowercase on the SDK folder to assign with the value you specified in meta.json

**Comment by irorochad** [comment]
Wow!

Thank you so much!
